.ci/integration_test.groovy

```diff
@@ -110,14 +110,14 @@ def call(ghprbActualCommit, ghprbCommentBody, ghprbPullId, ghprbPullTitle, ghprb
 
     def label = "regression-test-tispark"
 
-    podTemplate(name: label, label: label, instanceCap: 10, idleMinutes: 10, containers: [
+    podTemplate(name: label, label: label, instanceCap  : 20, idleMinutes: 10, containers: [
```

Comment:
how about set pod in namespace `jenkins-tispark`.

`podTemplate(name: label, label: label, instanceCap  : 20, idleMinutes: 10, namespace: jenkins-tispark,`